test/core/entity/PaddingSemver.test.ts

```
@@ -8,4 +8,66 @@ describe('test/core/entity/PaddingSemver.test.ts', () => {
+    it('should pad multi-digit numbers', () => {
+      assert.equal(PaddingSemVer.paddingVersion(123), '0000000000000123');
+      assert.equal(PaddingSemVer.paddingVersion(999999), '0000000000999999');
+    });
+
+    it('should handle 16-digit number', () => {
+      assert.equal(PaddingSemVer.paddingVersion(1234567890123456), '1234567890123456');
+    });
+
+    it('should throw for number exceeding 16 digits', () => {
+      assert.throws(() => {
+        PaddingSemVer.paddingVersion(12345678901234567);
```

Biome (2.4.4)

[error] This number literal will lose precision at runtime.

(lint/correctness/noPrecisionLoss)